requirements/requirements.shipit.txt

```diff
@@ -0,0 +1,2 @@
+bump2version
```

Comment:
some minor concern of adding another requirements file just for release, but i guess it is does separate things

Reply:
Agreed. I'm not sure the best place for the requirement. I mostly want to include it in a requirements file _somewhere_ for automation at some point in the future.

pulpcore and other pulp plugins that use the plugin-template stuff only specify it as a requirement
in the .ci stuff  (https://github.com/pulp/pulpcore/blob/master/.ci/assets/release_requirements.txt ) but
we aren't ported to that yet.

We don't have a general purpose requirements.txt either. Since it isn't needed at app runtime, just at devel/release time and we don't have a dev_requirements.txt etc either.